### PR TITLE
Include configs and logs for neutron HA

### PR DESCRIPTION
**Pull request by toabctl**
When using l3-HA, Neutron stores keepalived config files and log files
in /var/lib/neutron/ha_confs so collect these files, too.

### CODE REVIEW

File: suse_openstack_cloud [hunk 1]
```diff
@@ -359,6 +359,9 @@ for svc in keystone rabbitmq glance swift nova cinder neutron heat ceilometer \
         rabbitmq)
             pconf_files /etc/rabbitmq/enabled_plugins
             ;;
+        neutron)
+            pconf_files /var/lib/neutron/ha_confs/
```
Comment (aspiers) [inline suse_openstack_cloud]: `pconf_files` expects files as arguments, not directories.  I think you want `find_and_pconf_files`.  Please mark as untested if not tested yet ;-)

Comment (toabctl) [reply suse_openstack_cloud]: Fixed (at least the `find_and_pconf_files` part). Still needs testing (so I keep the WIP label for now)

**Comment by toabctl** [comment]
@KeithMnemonic did you test it as suggested by @aspiers ? It still had the WIP label due to that.

**Comment by KeithMnemonic** [comment]
Thomas I did not see the label, it had two approvals though. I guess we can revert it

**Pull request merged**